lua/lspconfig/util.lua

```diff
@@ -161,15 +161,16 @@ M.path = (function()
 
   -- Iterate the path until we find the rootdir.
   local function iterate_parents(path)
-    path = uv.fs_realpath(path)
     local function it(s, v)
-      if not v then
+      if not is_fs_root(v) then
```

Comment:
```suggestion
      if v and not is_fs_root(v) then
```
(to make it consistent w/ previous behavior when `path == nil`)